app/javascript/components/froggo-team-form.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <div>
+    <span>Nombre del equipo:</span><br><br>
+    <input type="text" v-model="team_name"><br><br>
+    <span>Agregar miembro:</span><br><br>
+    <users-dropdown
+        :users="users"
+        @UpdateSelected="updateSelected"
+    />
+   <div>
+    <button v-on:click="submitFroggoTeam()"> crear grupo </button>
+  </div>
+  </div>
+</template>
+
+<script>
+import UsersDropdown from './users-dropdown';
+import { CREATE_NEW_FROGGO_TEAM } from '../store/action-types';
+
+export default {
+
+  data() {
+    return {
+      team_name: "",
+      button_message: 'Crear grupo',
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
  Identifier 'button_message' is not in camel case                                  camelcase